### ISSUE TITLE
❌ Invalid Issue: images on pages are blurry and site not working properly

### ISSUE THREAD

**Issue by cbusron** [elementor/elementor#20173]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I woke up and my website is now blurry 

### Steps to reproduce

I have no idea

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently using the steps above.

### System Info

<html><body>

Operating System: | Linux |  
-- | -- | --
Software: | Apache |  
MySQL version: | Percona Server (GPL), Release 23, Revision 500fcf5 v5.7.23-23 |  
PHP Version: | 8.0.18 |  
PHP Memory Limit: | 512M |  
PHP Max Input Vars: | 1000 |  
PHP Max Post Size: | 516M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected

</body>
</html>

**Comment by nicholaszein** [comment]
Hey @cbusron, 

Thank you for submitting your issue. 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information.

* **Prerequisites:** please search the repository before submitting an issue, your issue is a **DUPLICATE** of #20162;
* **Description:** you did not provide a detailed description of your issue. We can't validate it without understanding the context;
* **Steps to replicate:** You did not provide any way for us to try to reproduce your issue;
* **System info:** You did not provide any information about your [System Info](https://elementor.com/help/systems-info/) as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

Without them, we simply can't help. 😕 

👉 Please follow the guidelines for opening a [new issue report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

### ⚠️ Note that GitHub issues are reserved for 🐞**Bug Reports**, and should not be used for support.

❔ If you have questions or need general support, please use one of the following channels:

* [Elementor Global Community](https://facebook.com/groups/Elementors/) on Facebook
* [Elementor Help Center](https://docs.elementor.com)
* [Support Forum](https://wordpress.org/support/plugin/elementor) on WordPress.org 
* [Elementor Developers Documentation](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Best regards
